src/components/SearchButton.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const SearchButton = () => {
+  return <button className="btn btn-primary">Search</button>;
+};
```

Comment:
Search button is nicely implemented! Looks like you still need to work on the search functionality though :)